app/models.py

```diff
@@ -2878,4 +2878,79 @@ class PhoneMessage(Base, ModelMixin):
     number = orm.relationship(PhoneNumber)
 
 
+class AdminAuditLog(Base):
```

Comment:
It doesn't have an `updated_at` that's why it doesn't inherit from the ModelMixin. An audit log that can be 'updated' is not much of an audit. We could add it and always have it be the same as a created_at but since this class doesn't need many of the methods in the ModelMixin we can pass without it. What do you think?

Reply:
Sounds good 👌